### ISSUE TITLE
Latest release yanked on PyPI

### ISSUE THREAD

**Issue by pordyna** [openPMD/openPMD-api#1705]
`0.16.0` is marked as yanked on pypi, so it is always ignored by pip except for when it is explicitly requested with `openpmd-api==0.16.0`. For me, it works anyway, at least when recompiling for MPI support. I didn't try the binary. 

Problem:
Pip will install older version instead and due to the old version requirements most likely also downgrade `numpy` to `1.*`.

**Comment by franzpoeschel** [comment]
Yup, I know. The first build for Pypi had some errors, so we yanked it. The errors are now patched, so this *could* be removed.
Unfortunately, this can't be fixed until January, because we apparently overlooked that only Axel has admin privileges for Pypi and he isn't back until then ¯\\\_(ツ)\_/¯

**Comment by pordyna** [comment]
Haha ok, just wanted to make sure you are aware of it.

**Comment by ax3l** [comment]
THanks again for the report!

We also made sure now that Franz can yank (admin) and push new (already before) releases to PyPI :)
